style.css

```diff
@@ -0,0 +1,574 @@
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+}
+
+.portfolio h2 {
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  padding-left: 24px;
+  padding-bottom: 12px;
+  margin-top: 0;
+  margin-bottom: 0;
+  color: aliceblue;
+}
+
+.portfolio p {
+  width: 327px;
+  height: 123px;
+  padding-left: 24px;
+  color: #ffff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.icons {
+  width: 200px;
+  height: 40px;
+  padding: 24px;
+  padding-top: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+main {
+  background: white;
+}
+
+.title {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', serif;
+  font-size: 40px;
+  font-weight: 400;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 120px;
+  margin-bottom: 0;
+}
+
+.vector {
+  padding-left: 24px;
+  padding-right: 24px;
+  align-items: start;
+}
+
+.placeholder {
+  padding-top: 24px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.work {
+  padding-left: 24px;
+  margin-bottom: 0;
+  height: 36px;
+  width: 327px;
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+}
+
+.paragraph {
+  height: 147px;
+  width: 327px;
+  left: 0;
+  top: 48px;
+  padding-left: 24px;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.teclast {
+  display: flex;
+  width: 314px;
+  height: 40px;
+  gap: 12px;
+  padding-left: 24px;
+}
+
+.css-button {
+  background: white;
+  width: 31px;
+  height: 20px;
+  padding: 10px, 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.html-button {
+  background: white;
+  width: 67px;
+  height: 20px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.boot-button {
+  background: white;
+  width: 95px;
+  height: 20px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.ruby-button {
+  background: white;
+  width: 31px;
+  height: 20px;
+  padding: 10px, 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.orange-button {
+  display: flex;
+  width: 124px;
+  height: 48px;
+  padding: 24px;
+}
+
+.see-project {
+  height: 20px;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  background: #ff6b00;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 24px;
+  align-items: center;
+  color: #ffff;
+}
+
+.grid-container {
```

Comment:
- [ ] Please adhere to the Figma design because your design has some issues related to spacing which makes your design looks unprofessional or not look good to the eye. Kindly have a look at the below table to understand the problem in a better way through visualization.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-06 12-50-53](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/de7a9e27-e8c2-464f-9fdb-05c4c4af9437)


</td>

<td>

![Screenshot from 2023-06-06 12-52-35](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/458be796-35e4-49c7-bc36-fa95036ba881)


</td>
</tr>
</table>